client/components/UserList/components/UserFilter/UserFilter.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+
+class UserFilter extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      initialState: []
```

Comment:
name is single but array means it should be plural. Also state name is not self explanatory